utils/dotcms/utilities.js

```diff
@@ -29,6 +29,7 @@ const DOTCMS_DOWN = 'DOTCMS_DOWN';
 const DOTCMS_NO_LAYOUT = 'DOTCMS_NO_LAYOUT';
 const DOTCMS_NO_AUTH = 'DOTCMS_NO_AUTH';
 const DOTCMS_CUSTOM_ERROR = 'DOTCMS_CUSTOM_ERROR';
+const LANG_COOKIE_NAME = 'dotSPALang';
```

Comment:
why not keep consistency name with previous `const` --> `DOTCMS_SPA_LANG` ?

Reply:
`DOTCMS_SPA_LANG` can mean anything but `LANG_COOKIE_NAME` is 100% attach to the use of this string which is the cookie.